### ISSUE TITLE
[Suggestion] Download Maps from withing PolyMC

### ISSUE THREAD

**Issue by splice11** [PolyMC/PolyMC#525]
### Role

I often download and play parkour maps from sites like minecraftmaps.com.

### Suggestion

I want it to work like the "Download mods" feature but for maps.

### Benefit

so you can download maps conveniently and quickly!

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

**Comment by DioEgizio** [comment]
that site needs an api first to this being possible.

**Comment by glowiak** [comment]
And it's not only that site. There are many other sites with maps, like http://minecraftmapy.pl

**Comment by splice11** [comment]
Should I close the issue then?

**Comment by Scrumplex** [comment]
Nah, just keep it open. We have a similar issue for shaderpacks (#170) and for resource packs (#256), so this one fits as well. Even if there is no easy solution for this now, doesn't mean we wont be able to implement this in the future.

**Comment by AshtonSnapp** [comment]
I know CurseForge has a category for worlds/maps, so you should be able to re-use the CurseForge API for downloading mods and use it to download maps as well. Assuming I'm understanding things correctly.
